requirements.txt

```diff
@@ -7,26 +8,34 @@ click==7.1.2
 distlib==0.3.1
 filelock==3.0.12
 Flask==1.1.2
+Flask-Migrate==2.5.3
+Flask-SQLAlchemy==2.4.4
 gunicorn==20.0.4
-identify==1.5.5
+identify==1.5.2
```

Comment:
Just curious - why have some of the packages been brought down in version number?

Reply:
I suspect it's because another package relies on earlier versions of these packages